lib/services/clusterconfig.go

```diff
@@ -144,44 +156,20 @@ func GetClusterConfigSchema(extensionSchema string) string {
 	} else {
 		clusterConfigSchema = fmt.Sprintf(ClusterConfigSpecSchemaTemplate, ","+extensionSchema)
 	}
+	clusterConfigSchema += `, "override": ` + fmt.Sprintf(ClusterConfigOverrideSchemaTemplate, "")
```

Comment:
Why is this `"override":` not a part of the clusterConfigSchema string? Otherwise it's not clear that it has override just looking that that var.

Reply:
I don't understand. Are you saying to make the variable name `clusterConfigSchema` more descriptive?

Reply:
No, sorry. I rather was wondering why not make `"override"` field just a part of `ClusterConfigSpecSchemaTemplate` string instead of appending here, something like:

```
const ClusterConfigSpecSchemaTemplate = `{
	"type": "object",
	"additionalProperties": false,
	"properties": {
	  "session_recording": {
		"type": "string"
	  },
	  "proxy_checks_host_keys": {
		"type": "string"
	  },
	  "cluster_id": {
		"type": "string"
	  },
	  "client_idle_timeout": {
		"type": "string"
	  },
	  "session_control_timeout": {
		"type": "string"
	  },
	  "disconnect_expired_cert": {
		"anyOf": [{"type": "string"}, { "type": "boolean"}]
	  },
	  "keep_alive_interval": {
		"type": "string"
	  },
	  "keep_alive_count_max": {
		"type": "number"
	  },
	  "local_auth": {
		"anyOf": [{"type": "string"}, { "type": "boolean"}]
	  },
	  "audit": {
		"type": "object",
		"additionalProperties": false,
		"properties": {
		  "type": {
			"type": "string"
		  },
		  "region": {
			"type": "string"
		  },
		  "audit_events_uri": {
			"anyOf": [
			  {"type": "string"},
			  {"type": "array",
			   "items": {
				 "type": "string"
			   }
			  }
			]
		  },
		  "audit_sessions_uri": {
			"type": "string"
		  },
		  "audit_table_name": {
			"type": "string"
		  }
		}
	  },
          "override": %v%v
	}
  }`
```

I actually have a feeling the way it's appended right now does not make up a correct json resource unless I'm missing something, as it basically just appends it so you end up with something like 

```
{
	"type": "object",
	"additionalProperties": false,
	"properties": {
	   {{props}}
	  },
	}
  }, "override": {override schema}
```

No?

Reply:
I'm appending the `override` part in this way because some (un)marshaling has to be done for `ClusterConfigOverride` itself. This way we're able to use the same schema for the `override` section of `ClusterConfig` as well as for `spec` of `ClusterConfigOverride`.

IIRC I checked and the final concatenation should end up OK. An error would pop up with some `tctl` actions if the schema were incorrect.